src/apps/jsgeneric/jsgeneric.cpp

```diff
@@ -13,33 +13,101 @@ namespace ccfapp
   using namespace kv;
   using namespace ccf;
 
-  using GenericTable = ccf::Store::Map<nlohmann::json, nlohmann::json>;
+  using LogTable = ccf::Store::Map<int32_t, std::string>;
+
+  static JSValue js_print(
+    JSContext* ctx, JSValueConst this_val, int argc, JSValueConst* argv)
```

Comment:
`this_val` is unused

Reply:
Yes, this is effectively a static call, it doesn't need access to the console object which is its "this". It still needs to be there in the signature for the call to work though.